client/components/relatedWidget/AddButton.jsx

```diff
@@ -14,12 +14,21 @@ class AddButton extends React.Component {
           <a href>
```

Comment:
Missed this earlier but it looks like this line is causing an error to show up.  It's not breaking anything at the moment but I think removing `href` would fix it if you aren't using that attribute.
<img width="557" alt="Screen Shot 2021-04-21 at 4 09 22 PM" src="https://user-images.githubusercontent.com/71413187/115627303-01a3de80-a2bc-11eb-98b8-25b69bfdde93.png">